composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name":        "ezsystems/ezpublish-legacy",
+    "name":        "sport1online/ezpublish-legacy",
```

Comment:
sorry.. I accidentally pushed the PR Button on github without changing the master to sport1 :) pls ignore it